<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetCamera</class>
 <widget class="QDialog" name="SetCamera">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SetCamera</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>621</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
border:1px solid#999999;
border -radius:15px; 

}</string>
   </property>
   <property name="title">
    <string>相机</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>19</y>
      <width>601</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="radioButton_1">
       <property name="text">
        <string>CCD1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="text">
        <string>CCD2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_3">
       <property name="text">
        <string>CCD3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_4">
       <property name="text">
        <string>CCD4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_5">
       <property name="text">
        <string>CCD5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_6">
       <property name="text">
        <string>CCD6</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>621</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
border:1px solid#999999;
border -radius:15px; 

}</string>
   </property>
   <property name="title">
    <string>参数</string>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>591</width>
      <height>171</height>
     </rect>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>591</width>
       <height>171</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>设备名字</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>增益</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>像素比</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_gain"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_pixie"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_4"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_name"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>模式</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>曝光</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="lineEdit_expose"/>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="lineEdit_7"/>
      </item>
      <item row="3" column="3">
       <widget class="QLineEdit" name="lineEdit_8"/>
      </item>
      <item row="0" column="3">
       <widget class="QComboBox" name="comboBox"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>611</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
border:1px solid#999999;
border -radius:15px; 

}</string>
   </property>
   <property name="title">
    <string>拍摄</string>
   </property>
   <widget class="QLabel" name="label_picture">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>211</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>拍摄一次</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>连续拍摄</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>radioButton_1</sender>
   <signal>toggled(bool)</signal>
   <receiver>SetCamera</receiver>
   <slot>getCCD(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>33</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>SetCamera</receiver>
   <slot>onshot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>391</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>SetCamera</receiver>
   <slot>conshot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>393</x>
     <y>408</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>SetCamera</receiver>
   <slot>stop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>518</x>
     <y>356</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>SetCamera</receiver>
   <slot>saveset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>536</x>
     <y>411</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_gain</sender>
   <signal>editingFinished()</signal>
   <receiver>SetCamera</receiver>
   <slot>changegain()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>SetCamera</receiver>
   <slot>changemode(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>580</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>647</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_expose</sender>
   <signal>editingFinished()</signal>
   <receiver>SetCamera</receiver>
   <slot>changeexpose()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>601</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>647</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>getCCD(bool)</slot>
  <slot>onshot()</slot>
  <slot>conshot()</slot>
  <slot>stop()</slot>
  <slot>saveset()</slot>
  <slot>changegain()</slot>
  <slot>changeexpose()</slot>
  <slot>changepixel()</slot>
  <slot>changemode(int)</slot>
 </slots>
</ui>
